internal/static/integrations/kafka_exporter/kafka_exporter.go

```diff
@@ -147,14 +188,21 @@ func New(logger log.Logger, c *Config) (integrations.Integration, error) {
 		return nil, fmt.Errorf("zookeeper lag is enabled but no zookeeper uri was provided")
 	}
 
+	// 30 is the default value
+	if c.PruneIntervalSeconds != 30 {
```

Comment:
Do we need to print the warning if PruneIntervalSeconds is set to 0?

Reply:
if PruneIntervalSeconds is different from 30 then it was explicitly set in the config by the user and that should trigger a warning (even if it is set to 0)